values.yaml

```diff
@@ -3,54 +3,34 @@
 # Declare variables to be passed into your templates.
 image:
   repository: storageos/node
-  tag: 0.9.0
+  tag: latest
   pullPolicy: IfNotPresent
-storageclass:
-  name: fast
-  # pool: default
-  # fsType: ext4
 cluster:
+  join:
   # Either a clusterId or firstNode should be set so that the cluster can
   # bootstrap.  The clusterId can be generated by using the StorageOS CLI:
   #   `storageos cluster create`
-  # firstNode is the hostname or ip address of a single cluster node.  This node
+  # firstNode is the hostname or ip address of a cluster node.  This node
   # will bootstrap the cluster when StorageOS starts for the first time, but
   # after boostrap serves no purpose and does not introduce a single point of
   # failure.
-  # clusterId: 23f30683-0149-4de0-9b4a-dd3998535416
-  # firstNode: gke-test01-node-default-pool-478bef52-wljs
-  join:
+storageclass:
+  name: fast
+  pool: default
+  fsType: ext4
 api:
   secretName: storageos-api
   secretNamespace: default
-  # address should be set to the external service address/name as it needs to be
-  # accessible by the Kubernetes master.
-  address: http://storageos:5705
+  # secrets are namespace specific, create 1+N for every namespace.
+  address: storageosapi:5705
+  # address is used to generate the ApiAddress value in the secret. This
+  # updated later with the service ClusterIP which is not known yet.
   username: storageos
   password: storageos
 service:
-  name: storageos
-  type: LoadBalancer
-
-  ## List of IP addresses at which the StorageOS api is available
-  ## Ref: https://kubernetes.io/docs/user-guide/services/#external-ips
-  ##
-  externalIPs: []
-  loadBalancerIP: ""
-  loadBalancerSourceRanges: []
-  externalPort: 15705
+  name: storageosapi
```

Comment:
nit: I'd prefer we call the service just 'storageos', since it will also serve the UI